Makefile

```diff
@@ -51,7 +52,9 @@ test-unit:
 	pytest -m unit
 
 test-integration:
-	curl -qo ${REMOTE_TEST_SETUP_NAME} ${REMOTE_TEST_SETUP_URL}
+	python pytest -m integration
+
+test-remote: prepare
 	python ${REMOTE_TEST_SETUP_NAME} pytest -m integration
```

Comment:
Looks like I missed this line earlier. Should be `python -m pytest ${REMOTE_TEST_SETUP_NAME} integration`

Reply:
It should be ‘pytest -m integration’, I copy pasted it wrongly :D